ert_shared/services/_base_service.py

```diff
@@ -333,6 +329,9 @@ def wait_until_ready(self, timeout: Optional[float] = None) -> bool:
             return not (
                 self._conn_info is None or isinstance(self._conn_info, Exception)
             )
+
+        if isinstance(self._conn_info, TimeoutError):
+            self.logger.critical(f"startup exceeded defined timeout {self._timeout}s")
```

Comment:
reminder: should this not be the `timeout` passed as the argument, instead of `self._timeout`?

Reply:
Good point! This class don't really need `self._timeout`